pyupgrade.py

```diff
@@ -22,11 +36,31 @@
 from tokenize_rt import tokens_to_src
 from tokenize_rt import UNIMPORTANT_WS
 
+DotFormatPart = Tuple[str, Optional[str], Optional[str], Optional[str]]
+PercentFormatPart = Tuple[
+    Optional[str],
+    Optional[str],
+    Optional[str],
+    Optional[str],
+    str,
+]
+PercentFormat = Tuple[str, Optional[PercentFormatPart]]
+
+ListCompOrGeneratorExp = Union[ast.ListComp, ast.GeneratorExp]
+ListOrTuple = Union[ast.List, ast.Tuple]
+NameOrAttr = Union[ast.Name, ast.Attribute]
+
+if False:
```

Comment:
```suggestion
if False:  # pragma: no cover (mypy)
```

weird, maybe I'm wrong

Reply:
Want it in the coveragerc instead?

Reply:
Hmm locally I still got 100% coverage even without this

Reply:
Of course, locally doesn't matter ;)

Reply:
python2 doesn't seem to get 100% coverage though and I'm not sure why -- might be the peephole optimizer

Reply:
and I'd prefer here, I plan to blow away `.coveragerc` in all by repos with a "standard" copy instead of the current drift I'm maintaining

Reply:
sure

Reply:
Maybe `coverage` is panic routing?